mlir_graphblas/src/test/GraphBLAS/check_ops.mlir

```diff
@@ -16,22 +16,14 @@
 
 module {
 
-    // CHECK: func @transpose_wrapper_no_swap(%[[ARG0:.*]]: [[CSR_TYPE:tensor<.*->.*>]]) -> [[CSC_TYPE:tensor<.*->.*>]] {
-    func @transpose_wrapper_no_swap(%sparse_tensor: tensor<2x3xf64, #CSR64>) -> tensor<2x3xf64, #CSC64> {
-        // CHECK-NEXT: %[[ANSWER:.*]] = graphblas.transpose %[[ARG0]] {swap_sizes = false} : [[CSR_TYPE]] to [[CSC_TYPE]]
-        %answer = graphblas.transpose %sparse_tensor { swap_sizes = false } : tensor<2x3xf64, #CSR64> to tensor<2x3xf64, #CSC64>
+    // CHECK: func @convert_layout_wrapper(%[[ARG0:.*]]: [[CSR_TYPE:tensor<.*->.*>]]) -> [[CSC_TYPE:tensor<.*->.*>]] {
+    func @convert_layout_wrapper(%sparse_tensor: tensor<2x3xf64, #CSR64>) -> tensor<2x3xf64, #CSC64> {
+        // CHECK-NEXT: %[[ANSWER:.*]] = graphblas.convert_layout %[[ARG0]] : [[CSR_TYPE]] to [[CSC_TYPE]]
+        %answer = graphblas.convert_layout %sparse_tensor : tensor<2x3xf64, #CSR64> to tensor<2x3xf64, #CSC64>
         // CHECK-NEXT: return %[[ANSWER]] : [[CSC_TYPE]]
         return %answer : tensor<2x3xf64, #CSC64>
     }
 
-    // CHECK: func @transpose_wrapper(%[[ARG0:.*]]: tensor<2x3xf64, [[CSR64:#sparse_tensor.encoding<{.*->.*}>]]>) -> tensor<3x2xf64, [[CSR64]]> {
-    func @transpose_wrapper(%sparse_tensor: tensor<2x3xf64, #CSR64>) -> tensor<3x2xf64, #CSR64> {
-        // CHECK-NEXT: %[[ANSWER:.*]] = graphblas.transpose %[[ARG0]] {swap_sizes = true} : tensor<2x3xf64, [[CSR64]]> to tensor<3x2xf64, [[CSR64]]>
-        %answer = graphblas.transpose %sparse_tensor { swap_sizes = true } : tensor<2x3xf64, #CSR64> to tensor<3x2xf64, #CSR64>
-        // CHECK-NEXT: return %[[ANSWER]] : tensor<3x2xf64, [[CSR64]]>
-        return %answer : tensor<3x2xf64, #CSR64>
-    }
-
```

Comment:
Add a check here that goes from CSC->CSR.

Reply:
Added test.